cluster/v1/types.go

```diff
@@ -95,6 +95,10 @@ type ManagedClusterStatus struct {
 
 	// Version represents the kubernetes version of the managed cluster.
 	Version ManagedClusterVersion `json:"version,omitempty"`
+
+	// ClusterClaims represents cluster information that a managed cluster claims
```

Comment:
Indicate that this is written from the managed cluster (I assume it is) and provide a couple known examples please. 

Be sure to indicate that the set of claims is not uniform across a fleet, some claims can be vendor or version specific and may not be included in from all managed clusters.

Reply:
Doc updated.